packages/mdc-icon-button/_mixins.scss

```diff
@@ -182,14 +190,35 @@
     }
   }
 
-  &:disabled {
-    @include mdc-feature-targets($feat-color) {
-      @include mdc-theme-prop(color, text-disabled-on-light);
-    }
+  @include mdc-icon-button-if-disabled_ {
```

Comment:
We can use `mdc-icon-button-disabled-ink-color` mixin here.

Reply:
Sure - I can extract the color part out and leave the pointer events stuff under `mdc-icon-button-if-disabled_`